merge-cluster/Readme.md

```diff
@@ -17,166 +28,194 @@ Conduktor Proxy's merge cluster feature help single kafka clients can work with
 
 ### Step 1: Review the environment
 
-As can be seen from `docker-compose.yaml` the demo environment consists of the following:
+As can be seen from `docker-compose.yaml` the demo environment consists of two kafka clusters. Each cluster has:
 
-* Two kafka clusters. One cluster has:
   * A single Zookeeper Server
   * A 2 node Kafka cluster
-* A single Conduktor Proxy container
-* A Kafka Client container (this provides nothing more than a place to run kafka client commands)
+  * A single Conduktor Proxy container
+  * A Kafka Client container (this provides nothing more than a place to run kafka client commands)
 
 ### Step 2: Start the environment
 
-Start the environment with
+Start the environment using:
 
 ```bash
 docker-compose up -d 
 ```
 
+Note:  You might see the error message `! conduktor-proxy The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested`.
```

Comment:
warning rather than error? @ehumber 
I assumed this was when running on apple ARM processors, so is fine as it's emulated.